src/NITTA/Project.hs

```diff
@@ -162,10 +162,12 @@ writeImplementation prj@Project{pTargetProjectPath = prjPath, pInProjectNittaPat
 copyLibraryFiles prj = mapM_ (copyLibraryFile prj) $ libraryFiles prj
     where
         copyLibraryFile Project{pTargetProjectPath, pInProjectNittaPath, pLibPath} file = do
-            let fullNittaPath = pTargetProjectPath </> pInProjectNittaPath
-            source <- makeAbsolute $ joinPath [pLibPath, file]
-            target <- makeAbsolute $ joinPath [fullNittaPath, "lib", file]
-            createDirectoryIfMissing True $ takeDirectory target
+            let fullNittaPath = joinPath [pTargetProjectPath, pInProjectNittaPath]
+            source <- makeAbsolute $ normalise $ joinPath [pLibPath, file]
+            target <- makeAbsolute $ normalise $ joinPath [fullNittaPath, "lib", file]
+            directory <- makeAbsolute $ normalise $ joinPath [fullNittaPath, "lib", takeDirectory file]
```

Comment:
Also, can you import `normalise` explicitly or with qualified name?

Please, add small issue to grep our project and find other places when it is reasonable to `normalise` paths for new project members.